app/views/home/index.html.erb

```diff
@@ -1 +1 @@
-<%= render partial: "signup_form" %>
+<div class="map"></div>
```

Comment:
Will this be a merge conflict? Has master been pulled onto your branch recently?